doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1301,6 +1301,7 @@ When adding a file to a dataset, you can optionally specify the following:
 - A description of the file.
 - The "File Path" of the file, indicating which folder the file should be uploaded to within the dataset.
 - Whether or not the file is restricted.
+- Whether or not the file skips tabular ingest.
```

Comment:
This should probably say that it defaults to "true" (?).

Reply:
I added the phrase that if tabIngest is not specified then it defaults to true.